libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/TwilioMessage.cs

```diff
@@ -11,6 +11,14 @@ namespace Microsoft.Bot.Builder.Adapters.Twilio
     /// </summary>
     public class TwilioMessage
     {
+        /// <summary>
+        /// Gets or sets the Author of the message.
+        /// </summary>
+        /// <value>
+        /// The Author of the message.
+        /// </value>
```

Comment:
Make the `value` property of the documentation single line to be consistent with the rest of the documentation.